src/main/java/application/rest/v1/actions/ResolutionContext.java

```diff
@@ -447,11 +457,21 @@ public ResolvedValue resolve(String pattern) {
                         else {
                             isFullyResolved.set(false);
                             result.append(t.toString());
+                            // JUNI
```

Comment:
Let's discuss.  In what way does this break the UI?

Reply:
We discussed and decided to throw exception and remove the special case handling for kibana log view.